datafusion/functions/src/regex/mod.rs

```diff
@@ -28,12 +28,53 @@ make_udf_function!(
     REGEXP_REPLACE,
     regexp_replace
 );
-export_functions!((
-    regexp_match,
-    input_arg1 input_arg2,
-    "returns a list of regular expression matches in a string. "
-),(
-    regexp_like,
-    input_arg1 input_arg2,
-    "Returns true if a has at least one match in a string,false otherwise."
-),(regexp_replace, arg1 arg2 arg3 arg4, "Replaces substrings in a string that match"));
+
+pub mod expr_fn {
+    #[doc = "returns a list of regular expression matches in a string. "]
+    #[doc = r" Return $name(arg)"]
```

Comment:
`#[doc = r" Return $name(arg)"]`  looks a bit weird, maybe we can delete it.

Reply:
Done